lib/commands/access.js

```diff
@@ -20,35 +20,23 @@ const subcommands = [
 ]
 
 class Access extends BaseCommand {
-  static get description () {
-    return 'Set access level on published packages'
-  }
-
-  static get name () {
-    return 'access'
-  }
-
-  /* istanbul ignore next - see test/lib/load-all-commands.js */
-  static get params () {
-    return [
-      'registry',
-      'otp',
-    ]
-  }
-
-  static get usage () {
-    return [
-      'public [<package>]',
-      'restricted [<package>]',
-      'grant <read-only|read-write> <scope:team> [<package>]',
-      'revoke <scope:team> [<package>]',
-      '2fa-required [<package>]',
-      '2fa-not-required [<package>]',
-      'ls-packages [<user>|<scope>|<scope:team>]',
-      'ls-collaborators [<package> [<user>]]',
-      'edit [<package>]',
-    ]
-  }
+  static description = 'Set access level on published packages'
+  static name = 'access'
+  static params =[
```

Comment:
```suggestion
  static params = [
```

Reply:
Once linting is working the linter will catch these.